e2e/test/scenarios/collections/uploads.cy.spec.js

```diff
@@ -268,45 +268,41 @@ describe("permissions", () => {
     });
   });
 
-  it(
-    "should show you upload buttons if you have unrestricted access to the upload schema",
-    { tags: ["@external"] },
-    () => {
-      H.restore("postgres-12");
-      cy.signInAsAdmin();
+  it("should show you upload buttons if you have unrestricted access to the upload schema", () => {
+    H.restore("postgres-12");
```

Comment:
Shouldn't this test have the tag?

Reply:
I removed the tag from the one test and added it to the surrounding .describe()

Reply:
Ahhhh, gotcha.
Sorry I missed that.